serverless/package.json

```diff
@@ -8,19 +8,20 @@
   },
   "dependencies": {
     "bignumber.js": "^9.0.1",
```

Comment:
StellarSDK relays on `bignumber.js` v4. Using two different versions of the same library may cause side effects.

Reply:
The package manager should manage any issues here so I'm fine leaving as is. I manually include bignumber.js in loads of other projects and have never had any issues.